### PR TITLE
Update virt docs and change the default virt type to kvm, bridge to v…

### PR DESCRIPTION
**Pull request by opoplawski**
…irbr0

## Description

xen is mostly dead at this point.  kvm is pretty much the standard now.

## Behaviour changes

Old: Default virt type was `xenpv` and virt bridge `xenbr0`

New: Defauil virt type is `kvm` and virt bridge is `virbr0`

## Category

This is related to a:

- [ ] Bugfix
- [ ] Feature
- [ ] Packaging
- [X] Docs
- [ ] Code Quality
- [ ] Refactoring
- [ X Miscellaneous

## Tests

- [ ] Unit-Tests were created
- [ ] System-Tests were created
- [ ] Code is already covered by Unit-Tests
- [ ] Code is already covered by System-Tests
- [X] No tests required

### CODE REVIEW

File: cobbler/items/system.py [hunk 1]
```diff
@@ -232,7 +232,7 @@
         * ``depth``: int
         * ``server``: str - Inheritable
         * ``virt_path``: str - Inheritable
-        * ``virt_type``: str - Inheritable; One of "xenpv", "xenfv", "qemu", "kvm", "vmware", "openvz"
+        * ``virt_type``: str - Inheritable; One of "qemu", "kvm", "xenpv", "xenfv", "vmware", "vmwarew", "openvz" or "auto".
```
Comment (SchoolGuy) [inline cobbler/items/system.py:235]: This is a changelog. Could you please describe the change in the section for 3.4.0?

File: cobbler/remote.py [hunk 2]
```diff
@@ -3869,7 +3869,7 @@ def get_menu_as_rendered(
         return self.xmlrpc_hacks({})
 
     def get_random_mac(
-        self, virt_type: str = "xenpv", token: Optional[str] = None, **rest: Any
+        self, virt_type: str = "kvm", token: Optional[str] = None, **rest: Any
```
Comment (SchoolGuy) [inline cobbler/remote.py:3872]: Please add a note to the 3.4.0 section in the changelog for the XML-RPC API.

**Review by SchoolGuy — CHANGES_REQUESTED** [review]
Thank for you changing this to a more reasonable default. I would appreciate if you could add a towncrier fragment for the changlog.

File: cobbler/remote.py (hunk 2 above)
Comment (opoplawski) [reply cobbler/remote.py:3357]: I think I figured this out as well.

**Review by SchoolGuy — APPROVED** [review]

**Pull request merged**